includes/blocks/class-convertkit-block-form-builder-field.php

```diff
@@ -110,42 +110,6 @@ public function get_name() {
 
 	}
 
-	/**
-	 * Returns this block's Title, Icon, Categories, Keywords and properties.
-	 *
-	 * @since   3.0.0
-	 *
-	 * @return  array
-	 */
-	public function get_overview() {
```

Comment:
This method isn't needed, as all field blocks extend this class. It isn't directly used to register a block.